### PR TITLE
Choropleth files cleanup

### PR DESCRIPTION
**Pull request by chejennifer**
- delete the old choropleth tool files
- remove unused choropleth endpoints & consolidate the choropleth endpoints for map and place page into one

### CODE REVIEW

File: server/__init__.py [hunk 1]
```diff
@@ -76,11 +76,10 @@ def create_app():
     app.register_blueprint(ranking.bp)
     app.register_blueprint(redirects.bp)
     app.register_blueprint(tools.bp)
-    from routes.api import (browser as browser_api, chart, choropleth, place as
+    from routes.api import (browser as browser_api, choropleth, place as
```
Comment (shifucun) [inline server/__init__.py:79]: shall we rename all the "choropleth" as "map"?

**Review by shifucun — APPROVED** [review]
Thanks for the clean up!

File: server/__init__.py (hunk 1 above)
Comment (chejennifer) [reply server/__init__.py:79]: I can, but would I want to rename choropleth stuff in the place page to map as well? Or we could keep this api as choropleth and the name "map" would just be specific to the map explorer tool which plots a choropleth map

Comment (beets) [reply server/__init__.py:79]: +1 to keeping map as just the product name

**Review by beets — APPROVED** [review]
nice, thanks for the cleanup

File: server/__init__.py (hunk 1 above)
Comment (shifucun) [reply server/__init__.py:79]: Change all the place page would be too many.. Yea, non-place change sounds good. This is not a must have, can do after all the feature is done.

**Pull request merged**